setup.py

```diff
@@ -28,7 +28,7 @@
     install_requires=[
         'requests>=2.7.0',
         'six',
-        'websocket-client==0.37.0'
+        'websocket-client'
```

Comment:
It looks like there were some fixes here that resolved incompatibility issues with newer version of websocket-client. We could check out if we still need this restriction or not.